_plugins/generators/onsite_groups_generator.rb

```diff
@@ -2,6 +2,17 @@ module Jekyll
   class OnsiteGroupsGenerator < Generator
 
     def generate(site)
+      # Get the meeting group permalinks
+      meeting_group_permalinks = Hash.new
+      site.
+        collections['onsite_groups'].docs.each do |group|
+          if !group.data['meetings'].nil?
```

Comment:
This is the same as the following in Ruby, which is a little more readable:

```rb
if group.data['meetings'] #...
```

Rails (really, Active Support) also brings us `present?` and `blank?` methods, which are pretty awesome (and Jekyll uses active support, so it is available).

Reply:
This may be worth skimming: https://blog.appsignal.com/2018/09/11/differences-between-nil-empty-blank-and-present.html